server/src/main/java/com/thoughtworks/go/server/service/PluginProfilesService.java

```diff
@@ -59,13 +60,15 @@ public M findProfile(String id) {
         return result;
     }
 
-    private void validatePluginProperties(PluginProfileCommand command, PluginProfile newPluginProfile) {
+    void validatePluginProperties(PluginProfileCommand command, PluginProfile newPluginProfile) {
         try {
             ValidationResult result = command.validateUsingExtension(newPluginProfile.getPluginId(), newPluginProfile.getConfigurationAsMap(true));
             addErrorsToConfiguration(result, newPluginProfile);
-        }catch (RecordNotFoundException e) {
+        } catch (RecordNotFoundException e) {
```

Comment:
If this exception's no longer for #6087 , then the message should change or remove the catch block for RecordNotFoundException?

Reply:
@varshavaradarajan -- it is used [here](https://github.com/gocd/gocd/blob/master/plugin-infra/go-plugin-access/src/main/java/com/thoughtworks/go/plugin/access/PluginRequestHelper.java#L40-L42)